### PR TITLE
Stylehint rules (part 3)

### PR DESCRIPTION
**Pull request by ianmcburnie**
Separated LESS and CSS stylelint configuration.

Enabled property ordering and indentation of 4 spaces for LESS and fixed all relevant violations of those rules. These rules are ignored for CSS (because the less compiler gives us no control over the output format).

### CODE REVIEW

File: docs/static/ds6/skin.css [hunk 1]
```diff
@@ -1190,13 +1190,13 @@ span.combobox__icon {
   border-color: transparent;
 }
 .page-notice {
+  -webkit-box-align: stretch;
+      -ms-flex-align: stretch;
+          align-items: stretch;
```
Comment (seangates) [inline docs/static/ds6/skin.css]: Do we like to have these "lined up" like this? Does the linter not care?

Comment (seangates) [reply docs/static/ds6/skin.css]: Ah, this is part of the dist. Never mind.

Comment (ianmcburnie) [reply docs/static/ds6/skin.css]: Yeah, autoprefixer does this formatting. It might be configurable. I'll check.

Comment (seangates) [reply docs/static/ds6/skin.css]: No worries. If it's generated I'm not too worried about it.

**Pull request merged**